curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ef9b03c81a63668521804e9.md

```diff
@@ -7,11 +7,11 @@ dashedName: step-64
 
 # --description--
 
-Make the text `freeCodeCamp.org` into a link by enclosing it in an anchor (`a`) element. The `href` attribute should be set to `https://www.freecodecamp.org`.
+Enclose below text `freeCodeCamp.org` with an anchor (`a`) element to make it a link and set the `href` attribute of `a` element to `https://www.freecodecamp.org`.
```

Comment:
```suggestion
Turn the existing `freeCodeCamp.org` text into a link by enclosing it in an anchor (`a`) element. The `href` attribute should be set to `https://www.freecodecamp.org`."
```

I think your new wording is slightly awkward. My suggestion is @moT01 's wording from the issue.

Reply:
ok. I'll add @moT01 's wording.